src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Noticia+Text:ital,wght@0,700;1,400;1,700&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 5px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 10px;
+  padding-right: 10px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-style: normal;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0%;
+  left: 0%;
+  height: 4px;
+  width: 37px;
```

Comment:
It would make sense to replace this with `right: 0`, so it would work with any link.